content/news/045/index.md

```diff
@@ -336,6 +336,48 @@ hardware ray tracing and glTF processing in Rust.
 [Ash]: https://github.com/ash-rs/ash
 [glrf-2-0]: https://www.khronos.org/gltf
 
+### [Luminol][luminol]
+
+![tile map editor and other tool windows like command event editor](luminol.png)
+
+[Luminol][luminol] by [@speak2erase] and [@somedevfox] is a remake
+of the RPG Maker editor, based mostly off of [RPG Maker XP][RMXP]
+(aka RMXP), with the intent of creating a more modern, feature rich,
+and open source version of RMXP.
+
+RGSS, RMXP's runtime, has already been open sourcecd thanks to [mkxp].
```

Comment:
I just noticed a spelling mistake in "sourcecd". How are these usually corrected?